environment.yml

```diff
@@ -0,0 +1,13 @@
+name: quantuminspire
+channels:
+  - conda-forge
+dependencies:
+  - pip
+  - numpy==1.16.3
+  - pytest==4.4.1
+  - matplotlib>=2.1
+  - pip:
+    - coreapi==2.3.3
+    - quantuminspire==0.7.0
+    - qiskit==0.8.1
```

Comment:
does this now mean that we have to maintain version of 3rd party libraries in two places? I.e., setup.py and environment.py? Mybinder claims to support requirements.txt and setup.py, can we not use that?

Reply:
I'll change it to use a requirements.txt. I'll put the dependencies needed for DEM850 also in.